### PR TITLE
Finish wiring up edit comments

### LINKED ISSUE #205: Edit comments
- Check if viewer can edit
- Show "..." icon on comments
- Alert controller with options (e.g. Edit)
- Use Slack TV for editing

### PR DESCRIPTION
**Pull request by rnystrom**
Looks like this is working! Going to send out a TestFlight build w/ this. Will merge tomorrow!

Fixes #205

### CODE REVIEW

**Comment by rnystrom** [comment]
Screenshots of it in action:

edit-workflow

File: Classes/Issues/Comments/IssueCommentSectionController.swift [hunk 1]
```diff
@@ -96,29 +100,43 @@ AttributedStringViewIssueDelegate {
         }
     }
 
-    func edit(sender: UIView) -> UIAlertAction? {
+    func editAction() -> UIAlertAction? {
         guard object?.viewerCanUpdate == true else { return nil }
-        return UIAlertAction(title: NSLocalizedString("Edit", comment: ""), style: .default, handler: { [weak self] _ in
-            guard let markdown = self?.object?.rawMarkdown,
+        return UIAlertAction(title: NSLocalizedString("Edit Comment", comment: ""), style: .default, handler: { [weak self] _ in
```
Comment (BasThomas) [inline Classes/Issues/Comments/IssueCommentSectionController.swift]: 👍

File: Classes/Issues/Comments/IssueCommentSectionController.swift [hunk 2]
```diff
@@ -283,6 +302,7 @@ AttributedStringViewIssueDelegate {
         alert.addActions([
             shareAction(sender: sender),
             deleteAction(),
+            editAction(),
```
Comment (Sherlouk) [inline Classes/Issues/Comments/IssueCommentSectionController.swift]: In my head it makes more sense to have the edit action before delete? Anyone else agree?

Comment (BasThomas) [reply Classes/Issues/Comments/IssueCommentSectionController.swift]: Hmm yeah, I feel like the delete should at least not be in the middle.

**Comment by rnystrom** [comment]
Man I wish PR review stuff worked. I can see there is activity but idk what it is. Will read comments later on laptop.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by BasThomas** [comment]
Yeah, it sucks. In those cases I open in SafariVC, but it's super annoying :/

File: Classes/Issues/Comments/IssueCommentSectionController.swift (hunk 2 above)
Comment (rnystrom) [reply Classes/Issues/Comments/IssueCommentSectionController.swift]: Oh ya great call

**Comment by rnystrom** [comment]
Note to handle #599

**Comment by randy3k** [comment]
Great. Look forward to it. Is there any timeline for a new release?

**Comment by rnystrom** [comment]
@randy3k I’m gonna try to test everything thoroughly over the next few days and submit a new version Friday!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by randy3k** [comment]
Excellent!

**Comment by randy3k** [comment]
Test

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Pull request merged**